src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveResourceService.cs

```diff
@@ -89,15 +89,14 @@ public async Task<RetrieveResourceResponse> GetInstanceResourceAsync(RetrieveRes
                 message.AcceptHeaders);
 
             string requestedTransferSyntax = validAcceptHeader.TransferSyntax.Value;
-            bool isOriginalTransferSyntaxRequested = DicomTransferSyntaxUids.IsOriginalTransferSyntaxRequested(requestedTransferSyntax);
+            bool isAnyTransferSyntaxRequested = DicomTransferSyntaxUids.IsAnyTransferSyntaxRequested(requestedTransferSyntax);
 
             if (message.ResourceType == ResourceType.Frames)
             {
                 return await GetFrameResourceAsync(
                     message,
                     partitionKey,
                     requestedTransferSyntax,
-                    isOriginalTransferSyntaxRequested,
```

Comment:
Simplifies method signatures by doing an evaluation at the end of the call chain rather than passing the value down.